app/src/main/java/org/fossasia/openevent/dbutils/DatabaseOperations.java

```diff
@@ -364,7 +364,9 @@ public Speaker getSpeakerById(int id, SQLiteDatabase mDb) {
         trackSelected = tracksCursor.getInt(tracksCursor.getColumnIndex(DbContract.Speakers.ID));
 
         //Select columns having track id same as that obtained previously
-        String sessionColumnSelection = DbContract.Sessions.TRACK + EQUAL + trackSelected;
+        String sessionColumnSelection = DbContract.Sessions.TRACK + EQUAL +
+                DatabaseUtils.sqlEscapeString(String.valueOf(trackSelected));
+        ;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [An empty statement (semicolon) not part of a loop](https://www.codacy.com/app/triplez/open-event-android/file/2406648853/issues/source?bid=3233850&fileBranchId=3233901#l369)